addon/components/em-form-group.js

```diff
@@ -98,8 +98,19 @@ export default Ember.Component.extend(InFormMixin, HasPropertyMixin, HasProperty
       return this.get('errors.firstObject.message') || this.get('errors.firstObject') || this.get('text');
     }
   }),
-  init() {
-    return this._super(...arguments);
+  hasSetForm: false,
+  didReceiveAttrs(arg) {
+    this._super(...arguments);
+    if(!!arg.newAttrs.form && !this.get('hasSetForm')){
+      this.set('form', arg.newAttrs.form.value);
```

Comment:
I don't understand why you use this line. You set a new form if there is already one but why ?

I'm working on tests and it fails because of this line.